python/pyspark/testing/utils.py

```diff
@@ -324,13 +325,17 @@ def assertDataFrameEqual(
     A util function to assert equality between `actual` (DataFrame) and `expected`
     (DataFrame or list of Rows), with optional parameters `checkRowOrder`, `rtol`, and `atol`.
 
+    Supports Spark, Spark Connect, and pandas-on-Spark DataFrames.
+    For more information about pandas-on-Spark DataFrame equality, see the docs for
+    `assertPandasOnSparkEqual`.
```

Comment:
nit: Can we add actual link for document

Reply:
this PR creates that doc, so won't link for now (but I can make a JIRA ticket to add the link later)